jest/private/jest_config_template.mjs

```diff
@@ -122,6 +123,15 @@ if (updateSnapshots) {
   config.snapshotResolver = bazelSnapshotResolverPath;
 }
 
+if (coverageEnabled) {
```

Comment:
Should this be enabled or disabled via a rule attribute?

Reply:
Does bazel not provide a way to do that already and `COVERAGE_OUTPUT_FILE` reflects that? Something like a tag to disable it?

Reply:
Yeah, I've learned that it's enabled by running `bazel coverge` and ensuring this target is instrumented in the `--instrumentation_filter`. Can be checked with `ctx.coverage_instrumented()`.